org.snowflake_ssh.App.json

```diff
@@ -0,0 +1,59 @@
+{
+	"app-id": "org.snowflake_ssh.App",
```

Comment:
`App` is NOT a good application name.

Reply:
`snowflake-ssh.org` doesn't exist.

Albeit you did acknowledge:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
Also according to upstream this is now called Muon.

Reply:
Sorry, I misunderstood. I thought he shouldn't be busy, English is not my native language. I'll fix it.